docs/pages/components/Combobox.mdx

```diff
@@ -0,0 +1,347 @@
+        value={value} 
+        onChange={handleChange}
+        onSelectionChange={handleSelectionChange}
+      >
+        <ComboboxOption value="Apple">🍎 Apple</ComboboxOption>
+        <ComboboxOption value="Banana">🍌 Banana</ComboboxOption>
+        <ComboboxOption value="Cucumber">🥒 Cucumber</ComboboxOption>
+        <ComboboxOption value="Orange">🍊 Orange</ComboboxOption>
+        <ComboboxOption value="Peach">🍑 Peach</ComboboxOption>
+        <ComboboxOption value="Pear">🍐 Pear</ComboboxOption>
+      </Combobox>
+    </FieldWrap>
+  );
+}
+```
+
+## Autocomplete
+
+There are a couple of different methods to control the auto completion of the options in the listbox. By default, autocomplete is set to "manual" meaning selections will be filtered based on the value of the input, and a value will need to be manually selected via click or keypress to change the value of the input field.
+
+### None
+
+When autocomplete is set to "none" the listbox will provide the full list of options and will not change based on the value of the input.
+
+```jsx example
+<FieldWrap>
+  <Combobox label="Auto Completion (None)" autocomplete="none">
+    <ComboboxOption>None</ComboboxOption>
+    <ComboboxOption>Manual</ComboboxOption>
+    <ComboboxOption>Automatic</ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+### Automatic
+
+When autocomplete is set to "automatic" the listbox will provide a filtered list of options with the first value active by default. The active value will become the value of the input when the listbox closes.
+
+```jsx example
+<FieldWrap>
+  <Combobox label="Auto Completion (Automatic)" autocomplete="automatic">
+    <ComboboxOption>None</ComboboxOption>
+    <ComboboxOption>Manual</ComboboxOption>
+    <ComboboxOption>Automatic</ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+## Props
+
+### Combobox
+
+<ComponentProps className={true} refType="HTMLDivElement" props={[
+  {
+    name: 'label',
+    type: ['string', 'number', 'ReactElement', 'ReactFragment', 'ReactPortal'],
+    required: true,
+    description: 'Label for the Combobox input field.'
+  },
+  {
+    name: 'options',
+    type: 'object',
+    description: 'Array of objects mapping to Combobox options. Alternatively, "ComboboxOption" can be used as children instead of this prop.'
+  },
+  {
+    name: 'value',
+    type: 'string',
+    description: 'Initial value to be applied to the combobox. Optionally used for "uncontrolled" comboboxes.'
+  },
+  {
+    name: 'defaultValue',
+    type: 'string',
+    description: 'Value to be applied to the combobox. Optionally used for "controlled" comboboxes.'
+  },
+  {
+    name: 'autocomplete',
+    type: ['none', 'manual', 'automatic'],
+    defaultValue: 'manual',
+    description: 'Sets the autocompletion type of the Combobox listbox.'
+  },
+  {
+    name: 'onSelectionChange',
+    type: 'function',
+    description: 'Callback function that gets invoked when the selected value is changed.'
+  },
```

Comment:
Should we document `onChange` here, too?

Reply:
☝️